docs/source/basics/messages.md

```diff
@@ -0,0 +1,32 @@
+# Messages
+
+Task executions produce logs, displayed on the Latch console to provide users visibility into their workflows. However, these logs tend to be terribly verbose. It's tedious to sift through piles of logs looking for useful signals; instead, important information, warnings, and errors should be prominently displayed. This is accomplished through the Latch SDK's new messaging feature.
```

Comment:
Can we also add an image here, so people know what to expect on Console when they use messages?